src/server/services/helpers/setFileIndex.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const indexFilePath = "./src/dataBase/file-indexData.json";
+
+let indexObject;
+let indexString;
+
+module.exports = function downloadIndex() {
+    const indexData = fs.readFileSync(indexFilePath);
+
+    try {
+        indexObject = JSON.parse(indexData);
+    } catch (error) {
+        console.error("Sorry, i can`t parse to JSON indexDATA file. \n" + error);
+    }
+
+    let currentFileIndex = indexObject.currentIndex;
```

Comment:
Wrong using of try-catch here. If smth went wrong and line 13 has been triggered and `indexObject` is undefined (line 4) but code goes down and you will have a problems ;)